python/cudf/cudf/core/series.py

```diff
@@ -390,6 +390,13 @@ def ndim(self):
         """
         return 1
 
+    @property
+    def size(self):
+        """
+        Return the number of elements in the underlying data.
+        """
+        return len(self)
```

Comment:
It looks like this should be implemented for DataFrame / MultiIndex / Index / Array as well, where maybe we push it to both `Frame` and `Column` classes?

Reply:
Added a `Frame.size` which is used by `DataFrame`/`Index` & `Series`. `MultiIndex` actually, so added a separate property for that.